client/src/index.js

```diff
@@ -27,57 +26,43 @@ const Routes = () => {
 
   return (
     <Router>
-      <Segment className="header-container" fluid>
-        <List horizontal floated="right" size="large">
-          <List.Item>
-            <Button as={Link} to="/">
-              Home
-            </Button>
-          </List.Item>
-          <List.Item>
-            <Button as={Link} to="/about">
-              FAQ
-            </Button>
-          </List.Item>
-
-          {isLoggedIn && (
-            <List.Item>
-              <Button as={Link} to="/add-new-profile">
-                New Profile
-              </Button>
-            </List.Item>
-          )}
+      <div className="absolute w-full">
+        <img
+          className="h-24 object-cover object-right w-full z-0 md:h-36"
+          src={MINLogo}
+          alt="MIN Logo"
+        ></img>
+      </div>
+      <nav className="flex flex-col justify-center h-24 ml-2 z-10 md:h-36 md:items-center">
+        <div className="h-12 flex space-x-4">
+          <Link to="/" className="nav-btn">
+            Home
+          </Link>
+          <Link to="/about" className="nav-btn">
+            FAQ
+          </Link>
           {localStorage.getItem("token") ? (
-            <List.Item name="logout">
-              <Button onClick={logout}>Logout </Button>
-            </List.Item>
+            <button onClick={logout} className="nav-btn">
```

Comment:
you can use the value for `isLoggedIn` here instead of getting the token from localStorage again?

Reply:
I noticed this but left it for now since it's not really relevant to these changes. It can be picked up in a task.